src/docmap-parser.ts

```diff
@@ -80,11 +80,30 @@ export type VersionedReviewedPreprint = ReviewedPreprint & {
   versionIdentifier: string,
 };
 
+export type Manuscript = {
+  doi?: string,
+  volume?: string,
+  eLocationId?: string,
+};
+
 export type ManuscriptData = {
   id: string,
+  manuscript?: Manuscript,
   versions: VersionedReviewedPreprint[],
 };
 
+const getManuscriptFromExpression = (expression: Expression): Manuscript | false => {
+  if (!expression.embodimentOf) {
+    return false;
+  }
+
+  return {
+    doi: expression.embodimentOf.doi,
+    volume: expression.embodimentOf.volumeIdentifier,
+    eLocationId: expression.embodimentOf.electronicArticleIdentifier,
```

Comment:
`expression.embodimentOf.doi`, `expression.embodimentOf.doi` and `expression.embodimentOf.electronicArticleIdentifier` are optional. We should expand the tests to account for this. This feels like it might fail or not return the expect value if one of the `embodimentOf` properties was not set.